packages/server/src/apollo.ts

```diff
@@ -39,7 +38,13 @@ export const connectApolloServer = async (app: IExpress) => {
     },
   })
 
-  server.applyMiddleware({ app })
+  server.applyMiddleware({
+    app,
+    cors: {
+      origin: [/^http:\/\/localhost/],
```

Comment:
Could this be replaced by the env variable? Since we only accept request from our own UI btm